src/styles/ui-select.less

```diff
@@ -0,0 +1,75 @@
+/* Fix some styles with the angular-ui-select bootstrap theme that isn't right when combined with all of our styles */
+.ui-select-bootstrap {
```

Comment:
Ok, I removed it for now and will follow up with a PR after moving it into common.